modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/VerifyBackupPartitionsTaskV2.java

```diff
@@ -83,6 +83,11 @@ public class VerifyBackupPartitionsTaskV2 extends ComputeTaskAdapter<VisorIdleVe
     /** First version of Ignite that is capable of executing Idle Verify V2. */
     public static final IgniteProductVersion V2_SINCE_VER = IgniteProductVersion.fromString("2.5.3");
 
+    /** Error thrown when idle_verify is called on an inactive cluster with persistence. */
+    public static final String IDLE_VERIFY_ON_INACTIVE_CLUSTER_ERROR_MESSAGE = "Can not perform the operation because " +
+        "the cluster is inactive and data regions are persistent. To activate the cluster call " +
+        "Ignite.cluster().state(ClusterState.ACTIVE).";
```

Comment:
This is control-utility command message, recomendation to use java code looks inappropriate here.

Reply:
Thank you, I will change to "To activate the cluster run the command: 'control.sh --set-state ACTIVE'."